integrationExamples/gpt/hello_world.html

```diff
@@ -23,16 +23,18 @@
                     sizes: [[300, 250]],
                 }
             },
-
             // Replace this object to test a new Adapter!
             bids: [{
-                bidder: 'appnexus',
```

Comment:
please don't change the base hello world example

Reply:
Done.